hotspot/src/cpu/aarch64/vm/macroAssembler_aarch64.cpp

```diff
@@ -1841,7 +1840,7 @@ void MacroAssembler::decrement(Register reg, int value)
   if (value < (1 << 12)) { sub(reg, reg, value); return; }
   /* else */ {
     assert(reg != rscratch2, "invalid dst for register decrement");
-    mov(rscratch2, (unsigned long)value);
+    mov(rscratch2, (u_int64_t)value);
```

Comment:
In general, it looks like this is far more frequently spelt `uint64_t` in the code base rather than `u_int64_t`.  I only find the latter in a few files in aarch64 specifically.  Any reason to use it instead?

Reply:
I matched the datatype of the prototype for the matching mov() here:
https://github.com/battleblow/openjdk-jdk8u/blob/bsd-port/hotspot/src/cpu/aarch64/vm/macroAssembler_aarch64.hpp#L452

Reply:
That's fair.  FWIW, I have to believe that `uint64_t` and `u_int64_t` are the same thing and that the presence of both in the code base only creates the potential for confusion :).  However, that is something to take up upstream rather than have our own diffs for, so this seems good.

Reply:
Thank you for the review.